### DISCUSSION TITLE
Maximum number of SRT client connections?

### DISCUSSION THREAD

**Discussion by JonnyTech** [bluenviron/mediamtx#3726] [Questions] [answered]
### Question

I am sending a single SRT stream with ffmpeg to MediaMTX to republish as SRT.

Then need to connect multiple clients (VLC, ffplay, OBS, etc) to view the stream.

Is there a maximum number of simultaneous connections/clients that MediaMTX supports?

Everything is on a fixed local network with no other traffic. Looking to connect 50 in total.

Since SRT is a UDP protocol, but ffmpeg only supports one connection, can MediaMTX handle it?

**Comment by aler9** [comment, marked as answer]
Hello, there is no maximum number of connections (unless you set `maxReaders`) and the bottleneck is almost always the LAN bandwidth. CPU and RAM consumption are minimal unless you are re-encoding the stream.

Regarding UDP, the server uses a single port for handling all SRT clients, there isn't any limit on that too.

**Comment by bilias** [comment]
I thought I shouldn't open a new issue for this.

Has anyone tested mediamtx for hundreds of clients for SRT or RTPM?

Does it scale to multiple cores/threads?

**Reply by JonnyTech** [reply]
I had approx 60 simultaneous SRT clients without issue.

**Reply by bilias** [reply]
Thanks for the reply.
